### PR TITLE
Enforce agent install requirements

### PR DESCRIPTION
**Pull request by jamonation**
## Type of change
documentation

### What should this PR do?
This PR adds a page about Enforce Agent install requirements.

### Why are we making this change?
We need a reference of the various requirements that a cluster must meet for Enforce Agent.

### What are the acceptance criteria? 
The page ought to list everything a private Enforce Agent needs.

### How should this PR be tested?
N/A

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title : "Chainguard Enforce Agent Installation Requirements"
+description: "Chainguard Enforce Agent Installation Requirements"
+type: "article"
+date: 2023-03-22 13:21:51 +0000 UTC
+draft: false
+images: []
+weight: 799
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md:8]: Maybe this should be higher up?

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+---
+title : "Chainguard Enforce Agent Installation Requirements"
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md:2]:
```suggestion
title : "Agent Installation Requirements"
```

Or "Enforce Agent"? I think we are trying to abbreviate the titles

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+description: "Chainguard Enforce Agent Installation Requirements"
+type: "article"
+date: 2023-03-22 13:21:51 +0000 UTC
+draft: false
+images: []
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md:7]: tags!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+weight: 799
+---
+
+The Chainguard Enforce Agent is installed into each of your Kubernetes clusters as a `StatefulSet` consisting of 2 pod relicas in their own namespace called `gulfstream`. The Enforce Agent is a statically compiled binary, requires minimal system resources,  and can be installed using a variety of tools. It supports a number of different Certified Kubernetes distributions and versions.
+
+If you are using a managed GKE or EKS cluster, consider installing Enforce in agentless (also referred to as SaaS) mode by following this guide: [Set Up Chainguard Enforce Cloud Account Associations](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/). This installation method also supports additional workload discovery like Google Cloud Run, AWS ECS, and AWS AppRunner, and does not consume any resources in your Kuberntes clusters.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md:13]:
```suggestion
If you are using a managed GKE or EKS cluster, consider installing Enforce in agentless (also referred to as SaaS) mode by following this guide: [Set Up Chainguard Enforce Cloud Account Associations](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/). This installation method also supports additional workload discovery like Google Cloud Run, AWS ECS, and AWS AppRunner, and does not consume any resources in your Kubernetes clusters.
```

**Review by ltagliaferri — APPROVED** [review]
Just a couple of comments on this, otherwise good to go

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md (hunk 3 above)
Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md:7]: Oh yeah, added!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md (hunk 1 above)
Comment (jamonation) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/installation-requirements.md]: I pushed it up with network requirements, nice catch.

**Pull request merged**